src/patternfly/layouts/Alignment/examples/centerAlign.hbs

```diff
@@ -0,0 +1 @@
+{{#> alignment pf-c-alignment--Modifier="pf-m-center" }} content {{/alignment}}
```

Comment:
I would make these blocks like this to better format them in the code block output.

```
{{#> alignment pf-c-alignment--Modifier="pf-m-center" }}
  content
{{/alignment}}
```

Reply:
I had it like that but the formatter doesn't like it :(

Reply:
Which formatter? Looks like the bullseye and gallery examples are formatted this way, but the rest of the layouts (and components) aren't.